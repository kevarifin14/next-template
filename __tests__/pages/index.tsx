import renderer from 'react-test-renderer';

import Index from 'pages/index';

describe('Index Page', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(<Index />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
