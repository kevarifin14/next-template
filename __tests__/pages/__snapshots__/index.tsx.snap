// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page matches snapshot 1`] = `
<div
  className="grid grid-cols-2 gap-4 px-4"
>
  <button
    className="inline-flex items-center justify-center border rounded-md shadow-sm focus:outline-none font-medium focus:ring-2 focus:ring-offset-2 text-white bg-primary-600 border-primary-600 hover:bg-primary-700 hover:border-primary-700 focus:outline-none    px-4 py-2 "
    type="button"
  >
    Primary
  </button>
  <button
    className="inline-flex items-center justify-center border rounded-md shadow-sm focus:outline-none font-medium focus:ring-2 focus:ring-offset-2  text-primary-700 bg-primary-100 border-primary-100 hover:bg-primary-200 hover:border-primary-200 focus:outline-none   px-5 py-3 text-lg "
    type="button"
  >
    Secondary
  </button>
</div>
`;
